synapse_invite_checker/rest/contacts.py

```diff
@@ -28,25 +28,20 @@
 from synapse_invite_checker.store import InviteCheckerStore
 from synapse_invite_checker.types import Contact
 
+# Version of the TiMessengerContactManagement interface. See:
+# https://github.com/gematik/api-ti-messenger/blob/main/src/openapi/TiMessengerContactManagement.yaml
+_TMCM_schema_version = "1.0.2"
 
-def invite_checker_pattern(path_regex: str, config: InviteCheckerConfig):
-    path = path_regex.removeprefix("/")
-    root = config.api_prefix.removesuffix("/")
-    raw_regex = f"^{root}/{path}"
+# This API prefix will probably be deprecated in the future
+CONTACT_MANAGEMENT_API_PREFIX = "/_synapse/client/com.famedly/tim/v1"
 
-    # we need to strip the /$, otherwise we can't register for the root of the prefix in a handler...
-    if raw_regex.endswith("/$"):
-        raw_regex = raw_regex.replace("/$", "$")
 
-    return [re.compile(raw_regex)]
-
-
-class InfoResource(RestServlet):
-    def __init__(self, config: InviteCheckerConfig, version: str):
+class ContactManagementInfoResource(RestServlet):
+    def __init__(self, config: InviteCheckerConfig):
         super().__init__()
         self.config = config
-        self.version = version
-        self.PATTERNS = invite_checker_pattern("$", self.config)
+        self.version = _TMCM_schema_version
+        self.PATTERNS = [re.compile(f"{CONTACT_MANAGEMENT_API_PREFIX}$")]
```

Comment:
I think having the prefix inline here makes it a lot harder to see, what the actually registered path is. I liked the previous approach more.